### PR TITLE
IDEMPIERE-5458 - Timezone defined by Organization

### PR DESCRIPTION
**Pull request by matheus-marcelinux**
https://idempiere.atlassian.net/browse/IDEMPIERE-5458

# Pull Request Checklist

## Checklist:

- [x] My code follows the [code guidelines](https://wiki.idempiere.org/en/Contributing_to_Trunk) of this project
- [x] My code follows the best practices of this project
- [x] I have performed a self-review of my own code
- [x] My code is easy to understand and review. 
- [x] In hard-to-understand areas, I have commented my code.
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] I have added unit tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings
- [ ] I have made corresponding changes to the documentation as follows:
- - [ ] New feature (non-breaking change which adds functionality): I have created the New Feature page in the project wiki explaining the functionality and how to use it. If relevant, I have committed sample data to the core seed to have usable examples in GardenWorld.
- - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected): I have documented the change in a clear way that everyone in the community can understand the impact of the change.
- - [ ] Improvement (improves and existing functionality): This documentation is needed if the improvement changes the way the user interacts with the system or the outcome of a process/task changes. If it is just, for instance, a performance improvement, documentation might not be needed. 
- [ ] The changed/added documentation is in the project wiki (not privately-hosted pdf files or links pointing to a company website) and is complete and self-explanatory.

### CODE REVIEW

File: migration/iD10/oracle/202211011331_IDEMPIERE-5458.sql [hunk 1]
```diff
@@ -0,0 +1,30 @@
+-- IDEMPIERE-5458
+SELECT register_migration_script('202211011331_IDEMPIERE-5458.sql') FROM dual;
+
+SET SQLBLANKLINES ON
+SET DEFINE OFF
+
+-- Nov 1, 2022, 1:31:02 PM BRT
+UPDATE AD_SysConfig SET Value='muriloht',Updated=TO_TIMESTAMP('2022-11-01 13:31:02','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_SysConfig_ID=50001
```
[the remark this answers is not in the document]
Comment (matheus-marcelinux) [reply migration/iD10/oracle/202211011331_IDEMPIERE-5458.sql]: Hi @CarlosRuiz-globalqss , done , can you send a new one ? Thanks !

**Review by hengsin — CHANGES_REQUESTED** [review]

File: migration/iD10/oracle/202211011331_IDEMPIERE-5458.sql [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+-- Nov 1, 2022, 1:31:26 PM BRT
+ALTER TABLE AD_OrgInfo ADD TimeZone VARCHAR2(60 CHAR) DEFAULT NULL 
+;
+
+-- Nov 1, 2022, 1:31:26 PM BRT
+ALTER TABLE AD_OrgInfo MODIFY AD_Org_ID NUMBER(10)
+;
+
+-- Nov 1, 2022, 1:31:26 PM BRT
+ALTER TABLE AD_OrgInfo MODIFY AD_Client_ID NUMBER(10)
```
[the remark this answers is not in the document]
Comment (hengsin) [reply migration/iD10/oracle/202211011331_IDEMPIERE-5458.sql:20]: hi @matheus-marcelinux , any update ?

[the remark this answers is not in the document]
Comment (matheus-marcelinux) [reply migration/iD10/oracle/202211011331_IDEMPIERE-5458.sql]: hi @hengsin, sorry for the late, done

**Comment by matheus-marcelinux** [comment]
> Missing migration script to add Time Zone field to Organization Info Tab.

Sorry @hengsin i was working on other stuff last weeks and forgot that. I'll create the missing script tomorrow

**Review by hengsin — CHANGES_REQUESTED** [review]
hi @matheus-marcelinux , please resolve the merge conflict.

@CarlosRuiz-globalqss , should the migration script for this goes into iD10 or 11 ?

**Comment by CarlosRuiz-globalqss** [comment]
> @CarlosRuiz-globalqss , should the migration script for this goes into iD10 or 11 ?

Yes, better to move to iD11.

Documentation is also pending

**Comment by matheus-marcelinux** [comment]
hi @CarlosRuiz-globalqss @hengsin i'll do that

**Review by hengsin — APPROVED** [review]
tested

**Pull request merged**